### ISSUE TITLE
Consider moving code maintenance to drupal.org

### ISSUE THREAD

**Issue by rvilar** [Ymbra/migrate_default_content#81]
Now that d.o is using gitlab to handle module repositories I think we should move issues and maintenance to that infrastructure and leave github.

I think it's better to use d.o to share with the community latest dev code using the same tools.

**Comment by rodrigoaguilera** [comment]
I agree and also I don't think we should wait for the next phases of the gitlab deployment for drupal.org.

I just did a synchronization of the commits from github to the new git.drupal.org so as far as I can recall the only thing left would be dealing with the open issues on github (moving them or deleting them).

**Comment by rvilar** [comment]
Oh yeah! Thank you for that.

We can keep this issue open until we remove or move all issues

**Comment by rodrigoaguilera** [comment]
There were different commits on the different mirrors (drupal.org and github) so I cherry picked from the github branch to the drupal.org and force pushed on github.

The issues here were moved to Drupal.org.

From my side this repo can be archived so we can work with the branches and the issues on drupal.org only.

@rvilar @nachosalvador please do the honors of archiving this github repo.
